Styles.css

```diff
@@ -0,0 +1,3 @@
+.Header1 {
+  color: blue;
+}
```

Comment:
- [ ] This `Header1` class name is not used in your HTML file (look at my comments before)